[datadog_team_notification_rule] Added resource and datasource (#3422)
DataDog/terraform-provider-datadog: docs/data-sources/team_notification_rules.md

---
# generated by https://github.com/hashicorp/terraform-plugin-docs
page_title: "datadog_team_notification_rules Data Source - terraform-provider-datadog"
subcategory: ""
description: |-
  Use this data source to retrieve information about Datadog team notification rules.
---

# datadog_team_notification_rules (Data Source)

Use this data source to retrieve information about Datadog team notification rules.

<!-- schema generated by tfplugindocs -->
## Schema

### Required

- `team_id` (String) The team ID to fetch notification rules for.

### Optional

> - `rule_id` (String) Optional rule ID to filter to a specific notification rule. If not provided, all notification rules for the team will be returned.

Reviewer: ```suggestion
- `rule_id` (String) Optional rule ID to filter to a specific notification rule. If not provided, all notification rules for the team are returned.
```
